config.js

```diff
@@ -5,7 +5,7 @@
 
 CKEDITOR.editorConfig = function( config ) {
 	// Define changes to default configuration here. For example:
-	// config.language = 'fr';
+	config.height = 500;
```

Comment:
The best way to avoid editing samples it's too add manual test right away - a little more work, but then it can be reused for the final PR version.